src/lightning_fabric/CHANGELOG.md

```diff
@@ -65,7 +65,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Removed
 
--
```

Comment:
more of a general question: do we really need a changelog here when this was never released?

Reply:
Since this is an evolution of Lite, yes. If it was completely new then I'd agree with you.